app/templates/views/email-branding/view-branding.html

```diff
@@ -0,0 +1,95 @@
+{% extends "views/platform-admin/_base_template.html" %}
+{% from "components/page-header.html" import page_header %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "components/branding-preview.html" import email_branding_preview %}
+
+{% set page_title = email_branding.name %}
+
+{% block per_page_title %}
+{{ page_title }}
+{% endblock %}
+
+
+{% block backLink %}
+{{ govukBackLink({ "href": url_for('.email_branding') }) }}
+{% endblock %}
+
+{% block platform_admin_content %}
+
+<div class="govuk-grid-row bottom-gutter">
+    <div class="govuk-grid-column-full">
+        {{ page_header(email_branding.name) }}
+
+        {{ email_branding_preview(email_branding.id) }}
+
+        <h2 class="heading-medium">Organisations using this branding as their default</h2>
+        <nav class="browse-list">
+            {% if branding_orgs %}
+                <ul>
+                    {% for organisation in branding_orgs %}
+                        <li class="browse-list-item">
+                            <a class="govuk-link govuk-link--no-visited-state browse-list-hint"
+                                href="{{ url_for('.organisation_settings', org_id=organisation.id) }}">
+                                {{ organisation.name }}
+                            </a>
+                        </li>
+                    {% endfor %}
+                </ul>
+            {% else %}
+                <p class="hint">This branding is not a default for any organisations.</p>
```

Comment:
```suggestion
                <p class="hint">No organisations use this branding as their default.</p>
```

Part of me wonders whether we could get away with something as simple as `None` for this and the services list below – but let’s stick with this for now.